spyglass/cli.py

```diff
@@ -156,6 +158,10 @@ def get_parser():
                              '  mhr90  - Mirror horizontal and rotate 90 CW\n'
                              '  r270   - Rotate 270 CW'
                         )
+    parser.add_argument('-tf', '--tuning_filter', type=str, default="", nargs='?', const="",
+                        help='Set a tuning filter file name.')
+    parser.add_argument('-tfd', '--tuning_filter_dir', type=str, default="", nargs='?',const="",
```

Comment:
This should be `default=None`, as you don't want an empty string as your default directory. Same goes for the file, but there it doesn't matter atm as you do, `if len(tuning_filter) > 0:`. That check could be changed to `if tuning_filter:` with `default=None`

Reply:
Done. I used empty string to keep it always string, but it makes more sense this way. Done.